env/terragrunt.hcl

```diff
@@ -3,14 +3,15 @@ locals {
 }
 
 inputs = {
-  account_id        = "${local.vars.inputs.account_id}"
+  account_id        = local.vars.inputs.env == "local"? "" : "${local.vars.inputs.account_id}"
```

Comment:
Not a breaking change or anything, but if you wanted to clean up logic in here, you could add all of these to the environment specific `env_var.hcl` files so that you didn't need these checks.  Your local `env_var.hcl` could just look something like:
```hcl
inputs = {
  env        = "local"
  domain     = ""
  account_id = ""
} 
```